### ISSUE TITLE
[Idea] Dark Mode

### ISSUE THREAD

**Issue by bluewingtitan** [marblear/issues#9]
Propably a pretty low priority, a test viewer mentioned he would like that.

**Comment by derwaldgeist** [comment]
Yep, that idea already came up when we swapped the original dark design with a bright one.
